waddrmgr/address.go

```diff
@@ -263,10 +287,26 @@ func (a *managedAddress) ExportPrivKey() (*btcutil.WIF, error) {
 	return btcutil.NewWIF(pk, a.manager.chainParams, a.compressed)
 }
 
+// newManagedPublicAddress returns a new address based on the passed account
+// and public key hash.
+func newManagedPublicAddress(m *Manager, account uint32, pkHash []byte) (*managedAddress, error) {
```

Comment:
Thanks, keeping consistent with `newScriptAddress` I think I'll go with `newPKHashAddress`. It's not really 'managed' so I've removed that.